providers/harvest/crawler.js

```diff
@@ -24,9 +24,36 @@ class CrawlingHarvester {
       method: 'POST',
       body,
       headers,
-      json: true
+      json: true,
+      resolveWithFullResponse: true
```

Comment:
I;m not sure the overall intent here. We ought not do console logs as no one will see those. In general the client of the service should get back a reasonable status code. We likely should take the response from the crawler and "do the right thing". In some cases that will be retry. In others it will be return the same error that we got (e.g, 400) and in others it might be a 500 if somehow we really got it wrong.

Reply:
I wasn't able to test end to end as I couldn't get the crawler to work but it did give me 500 unauthorized errors so assumed it was propagating back up to the caller. Will write some unit tests to dive in deeper now and ensure correct responses are being returned in the appropriate way in the given situation